packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -889,15 +890,12 @@ describe("AttributesController", function () {
         });
 
         describe("Validator for own shared identity attribute successions", function () {
-            afterEach(async function () {
-                const attributes = await consumptionController.attributes.getLocalAttributes();
-                for (const attribute of attributes) {
-                    await consumptionController.attributes.deleteAttributeUnsafe(attribute.id);
-                }
-            });
-
-            test("should catch if the source attributes do not succeed one another", async function () {
-                const predecessorRepo = await consumptionController.attributes.createLocalAttribute({
+            let predecessorRA: LocalAttribute;
+            let successorRA: LocalAttribute;
+            let predecessorOSIA: LocalAttribute;
+            let successorOSIAParams: IAttributeSuccessorParams;
```

Comment:
please do not use abbreviations (same for the other test files)

Reply:
In the context of Attribute successions, we use these abbreviations in many places. It is much less cluttered than writing e.g. successorOwnSharedIdentityAttributeParams.  I have a strong opinion against removing them in the test files.

Reply:
Yeah, I missed that. Has to be changed everywhere b/c this is VERY hard to process when you are not into the topic. Let's start by changing this in new code!

Reply:
I understand that we should try to use descriptive variable names. Nevertheless, we should aim at finding an appropriate balance between being expressive enough and keeping the code readable. So, I agree with you not to use abbreviations in the source code that can only be understood by someone who is deep into the topic. However, in tests where the context is known and where there are many repetitions, in my opinion it is legitimate to use abbreviations.
 
Also, I'd appreciate it not to use the imperative form in an ongoing discussion, in order to not create the impression that further arguments are not listened to.

Reply:
> Also, I'd appreciate it not to use the imperative form in an ongoing discussion, in order to not create the impression that further arguments are not listened to.

👍🏼 

> However, in tests where the context is known and where there are many repetitions

You can not hope or expect that the reader has read ALL the code and understands everything about it. I have to read this variable and understand what it's used for. I don't want to have a glossary to understand tests..